src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/Retry.scala

```diff
@@ -1,36 +1,34 @@
 package org.broadinstitute.dsde.rawls.dataaccess
 
+import akka.actor.ActorSystem
```

Comment:
refactored to do retries in Futures using the akka scheduler... no blocking sleeps